Introduction.md

```diff
@@ -0,0 +1,13 @@
+### Introduction
+Testing is an important aspect of coding, and generating data by hand is tedious.
+Thus, Faker, which generates data for you, is a widely used package to generate realistic fake data.
+Faker has been and is being developed in many different languages, including Perl (Original), Ruby, Python, PHP, and JavaScript.
```

Comment:
FakerJs (Faker) is a data generation tool that found its origins in languages such as Perl, Ruby, Python, and PHP.

//here you would mention that this version is for Javascript

Reply:
I feel as if these first three lines could be combined without losing much in the way of context. Stuff below looks good though!